src/plugins/data/public/now_provider/now_provider.ts

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
Would this make more sense inside of `utils/` since it's not exposing a service?